src/absil/illib.fs

```diff
@@ -503,9 +503,20 @@ module String =
     let uppercase (s: string) =
         s.ToUpperInvariant()
 
-    let isUpper (s: string) = 
-        s.Length >= 1 && Char.IsUpper s.[0] && not (Char.IsLower s.[0])
-        
+    // Scripts that distinguish between upper and lower case (bicameral) DU Discriminators and Active Pattern identifiers are required to start with an upper case character.
```

Comment:
Nit: `Scripts` - this comment implies that this change is only relevant for F# scripts. It's also relevant to regular .fs files, yes?

Reply:
@cartermp , scripts in this case means written form of a language.